src/platform_impl/windows/util.rs

```diff
@@ -26,6 +27,9 @@ use windows_sys::Win32::UI::WindowsAndMessaging::{
 
 use crate::window::CursorIcon;
 
+pub static WIN_VERSION: LazyLock<windows_version::OsVersion> =
+    LazyLock::new(windows_version::OsVersion::current);
+
```

Comment:
Just copy this:
https://github.com/tauri-apps/tao/blob/5cc92980c4b112125668e83c436524d53402e983/src/platform_impl/windows/util.rs#L430-L431